packages/react-query/package.json

```diff
@@ -12,11 +12,11 @@
   },
   "types": "build/lib/index.d.ts",
   "main": "build/lib/index.js",
-  "module": "build/lib/index.esm.js",
+  "module": "build/esm/index.js",
   "exports": {
     ".": {
       "types": "./build/lib/index.d.ts",
-      "import": "./build/lib/index.mjs",
```

Comment:
`mjs` is required for modern node versions and should be left as-is

Reply:
Emmm, in that case, a better build approach may be needed here.